awx/ui_next/src/screens/Inventory/Inventory.jsx

```diff
@@ -61,10 +62,14 @@ function Inventory({ history, i18n, location, match, setBreadcrumb }) {
   if (
     location.pathname.endsWith('edit') ||
     location.pathname.endsWith('add') ||
-    location.pathname.includes('groups/')
+    location.pathname.includes('groups/') ||
```

Comment:
For this line and the line below the cardHeader is generated elsewhere.  This is necessary because these are nested resources which require different card header/tabs than what is generated here.  We have to generate those different tabs elsewhere because we don't have required information in this component.